bciers/apps/reporting/src/app/components/context/CurrentReportingYearContext.tsx

```diff
@@ -0,0 +1,5 @@
+"use client";
```

Comment:
I'm not sure a client component to hold state is the best way to do this. We have a `ReportingYearService` (does the same as this, it's hardcoded to 2024 for now) that could be used to add an extra field on the `api/reporting/reports` API endpoint providing data for the Operations page instead, that would be less code and removing the need for managing this frontend state

Reply:
I was thinking about this too. I had written up a quick API on top of that service, but decided to go this way. I had assumed that since there would be other places that would use it (ie. bcgov/cas-reporting#238), it might be useful to use this as a start in a single place in the frontend, even if it ultimately fetched against `ReportingYearService` to initialize. Willing to be wrong in that thinking though and just fetch against the service directly where it's needed.